pkg/plugin/driver/base.go

```diff
@@ -31,6 +31,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// EnvKey is the env value expected to be set in order to inform the worker env about what mode it is in.
+	// Technically this is also discernable via the command invoked, but this places the
+	// information in the standard env location.
+	EnvKey = "DRIVER"
```

Comment:
I'm not certain I really need this. I originally thought I needed it but when starting up the sonobuoy worker we use a different command for each type of (global/node-specific) and _that_ is what triggers the different logic, not anything abou the URL.

I don't think this value is bad to have, but it does complicate the PR a bit. I think I'll remove it.